examples/models/file/daylight.py

```diff
@@ -1,3 +1,13 @@
+''' A plot shading the daylight hours in a city's local time (which are
+discontinuous due to summer time) using the Patch and Line plot elements to
```

Comment:
I suppose `Patch` and `Line` should be enclosed in double backticks, or do we have to use the full `:class:` syntax (/cc @bokeh/dev)?.

Reply:
Good catch!

Reply:
I put in the double backticks; let me know if you need a further change.